src/components/check-your-phone/index.njk

```diff
@@ -21,6 +21,9 @@
     <form action="/check-your-phone" method="post" novalidate="novalidate">
         <input type="hidden" name="_csrf" value="{{csrfToken}}"/>
         <input type="hidden" name="phoneNumber" value="{{phoneNumber}}"/>
+         {# These feilds are needed when the page is re-rendered once an error has occured #}
```

Comment:
```suggestion
         {# These fields are needed when the page is re-rendered once an error has occurred #}
```

Could correct this here, I'd also be up for moving the explanation out of a comment and into a commit message.
But feels like this is one where we want to move quickly and tidy up later.

So go with whatever makes sense to you right now!